src/diffusers/models/resnet.py

```diff
@@ -374,6 +438,76 @@ def forward(self, x):
         return x * torch.tanh(torch.nn.functional.softplus(x))
 
 
+class Conv1dBlock(nn.Module):
+    """
+    Conv1d --> GroupNorm --> Mish
+    """
+
+    def __init__(self, inp_channels, out_channels, kernel_size, n_groups=8):
+        super().__init__()
+
+        self.block = nn.Sequential(
```

Comment:
Could we please not use `nn.Sequential` here? It's hard to debug. 

Could you maybe instead write down the layers:

```python
self.conv = nn.Conv1d(...)
```

and `RearrangeDim` should not be a class - it should just be a function IMO

Reply:
@patrickvonplaten how do you go about matching the strings of the parameters in the pretrained model to the new names when sequential is removed?